package exceptionhandling;

public class exceptionhandling1 {
public static void main(String[] args) {
	
	
	
	System.out.println(1);
	System.out.println(2);
	System.out.println(3);
	System.out.println(4);
	try
	
	{
		System.out.println("next time we will correction");
	}
	 catch(ArithmeticException e)
	
	{
		   System.out.println(5/0);
 
	}
    System.out.println(6);
    System.out.println(7);
	
	
	
	
	
	
}




}
